Makefile

```diff
@@ -42,6 +42,13 @@ golint:
 	@gometalinter.v1 --install
 	@gometalinter.v1 --vendor --deadline=600s ./...
 
+# shellcheck target for checking shell scripts linting
+shellcheck: /tmp/shellcheck-latest/shellcheck
```

Comment:
instead of /tmp/shellcheck-latest/shellcheck have an end point. binshellcheck or getshellcheck?